packages/common/src/downloads/_internal/file-url-fetchers/fetchExportImageDownloadFile.ts

```diff
@@ -53,7 +53,6 @@ export async function fetchExportImageDownloadFile(
   // }
 
   const blob: Blob = await request(`${entity.url}/exportImage`, requestOptions);
-  progressCallback && progressCallback(DownloadOperationStatus.COMPLETED);
```

Comment:
As part of this PR, we made the decision that `fetchDownloadFile()` delegates are no longer responsible for emitting the "complete" event. It is up to the caller of `fetchDownloadFile()` to notify that the download is complete.